addons/l10n_ch/models/res_bank.py

```diff
@@ -332,14 +331,22 @@ def _is_qr_reference(self, reference):
                and re.match('\d+$', reference) \
                and reference == mod10r(reference[:-1])
 
-    def _eligible_for_qr_code(self, qr_method, debtor_partner, currency):
+    def _eligible_for_qr_code(self, qr_method, debtor_partner, currency, raises_error=True):
```

Comment:
instead of adding an extra parameter for that, what about using a sub-method instead?
`_l10n_ch_check_qr_code_config` returning a list of errors && `_l10n_ch_ensure_exigible_for_qr_code` raising the errors?